user/getting-started.md

```diff
@@ -89,6 +89,37 @@ $ echo "import foo" > sample.py
 $ bblfsh client sample.py
 ```
 
+### Overriding driver images
+
+Perhaps you need that Babelfish Server runs different driver images than the default ones. You can configure what images it should use throw the environment variable `BBLFSH_DRIVER_IMAGES`. The command `bblfsh server ` looks for this variable which must look like:
```

Comment:
"you need that **the** Balbelfish **s**erver...".

If would be probably better like: 

 "In case you need that the Babelfish server to run different driver images than the default ones..."